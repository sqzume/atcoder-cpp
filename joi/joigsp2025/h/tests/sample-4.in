5 20 100
0 3 1
20 2 2
40 1 3
60 4 4
80 2 5
11
0
50
100
150
200
250
300
350
400
450
500
