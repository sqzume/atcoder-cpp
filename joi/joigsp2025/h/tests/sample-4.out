6
8
10
12
13
15
16
18
19
20
20
